subsys/net/ip/net_shell.c

```diff
@@ -2168,6 +2168,8 @@ int net_shell_cmd_stacks(int argc, char *argv[])
 	       CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE, unused,
 	       CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE - unused,
 	       CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE, pcnt);
+#else
+	printk("Enable CONFIG_INIT_STACKS to see usage information.");
```

Comment:
This is missing \n at the end of the print.

Reply:
Fixed.